db/re/skill_db.yml

```
@@ -45018,6 +45018,37 @@ Body:
         - Level: 5
           Amount: 78
     Status: WeaponBlock_On
+  - Id: 6518
+    Name: CD_DIVINUS_FLOS
+    Description: Divinus Flos
+    MaxLevel: 5
+    Type: Magic
+    TargetType: Attack
+    DamageFlags:
+      Splash: true
+    Range: 9
```

```suggestion
    Flags:
      MultiHitOnPlants: true
    Range: 9
```
Currently negative hit count is not damaging plants, only normal values